src/bundle/Resources/public/scss/_bulk-operation-failed-modal.scss

```diff
@@ -0,0 +1,25 @@
+.ez-bulk-operation-failed-modal {
+    .modal-dialog {
+        max-width: 60vw;
+    }
+
+    .modal-header {
+        background-color: $ez-color-base-pale;
+
+        .modal-title {
+            font-size: calculateRem(24px);
+        }
+
+        .close {
+            margin-top: calculateRem(-9px);
+        }
+    }
+
+    .modal-body {
```

Comment:
Maybe you can use the Bootstrap utility classes instead? https://getbootstrap.com/docs/4.1/utilities/spacing/